source/assets/js/preview.js

```diff
@@ -0,0 +1,49 @@
+import $ from 'jquery';
+
+$(function() {
+	let initialVariant = window.location.hash.substr(1),
+		reservedIds = [/* 'changelog', 'documentation'*/],
+		selectVariant = () => {
+			let currentVariant = window.location.hash.substr(1),
+				$currentVariant = $('#' + currentVariant),
+				event;
+
+			if (!$currentVariant.length) {
+				return;
+			}
+
+			$currentVariant.prop('checked', true);
+
+			if (typeof window.CustomEvent === 'function') {
```

Comment:
Since we're using jQuery anyway, I guess we can just use `$.trigger`.

Reply:
I tried it several ways… but I cannot make it work. I seem to have forgotten how we built th client JS architecture :( Open to suggestions. Feel free to clone, fix and push, of course!